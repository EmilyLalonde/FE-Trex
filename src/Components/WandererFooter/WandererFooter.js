import React, { Component } from 'react';
import { 
  StyleSheet, 
  Text, 
  View, 
  TouchableOpacity
} from 'react-native';

export default class WandererFooter extends Component {

  constructor(props) {
    super(props)
  }
  
  render() {

    //pass user_id to each of the navigations

    return(
      <View style={styles.footer}>
            <TouchableOpacity
            
            onPress={() => this.props.navigate('MyTrips')}
            >
              <Text style={styles.footerText}>Trips</Text>
            </TouchableOpacity>
            <TouchableOpacity
            onPress={() => this.props.navigate('SafetyInfo')}
            >
              <Text style={styles.footerText}>Safety</Text>
            </TouchableOpacity>
            <TouchableOpacity
            
            onPress={() => this.props.navigate('MyFollowers')}>
              <Text style={styles.footerText}>Followers</Text>
            </TouchableOpacity>

          </View>
    )
  }
}


const styles = StyleSheet.create({
  footer: {
    backgroundColor: '#1C4263',
    flexDirection: 'row',
    padding: 10,
    justifyContent: 'space-around',
    height: 60
  }, 
  footerText: {
    color: 'white',
    fontSize: 20
  }
})