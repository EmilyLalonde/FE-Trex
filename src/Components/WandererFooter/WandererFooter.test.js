import React from "react";
import "react-native";
import WandererFooter from "./WandererFooter";
import { shallow } from "enzyme";

describe("WandererFooter", () => {
  it("should return true", () => {
    expect(true).toEqual(true);
  });
  it("should match the snapshot when loading", () => {
    let wrapper = shallow(<WandererFooter navigation={{ navigate: jest.fn() }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
