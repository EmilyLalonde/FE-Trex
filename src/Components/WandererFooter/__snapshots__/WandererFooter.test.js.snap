// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WandererFooter should match the snapshot when loading 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#1C4263",
      "flexDirection": "row",
      "height": 60,
      "justifyContent": "space-around",
      "padding": 10,
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 20,
        }
      }
    >
      Trips
    </Text>
  </TouchableOpacity>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 20,
        }
      }
    >
      Safety
    </Text>
  </TouchableOpacity>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 20,
        }
      }
    >
      Followers
    </Text>
  </TouchableOpacity>
</View>
`;
